pkgs/development/python-modules/biliass/default.nix

```diff
@@ -2,20 +2,20 @@
 , buildPythonPackage
 , fetchPypi
 , pythonOlder
-, protobuf
+, protobuf4_21
```

Comment:
This is not allowed in pythonPackages because it will cause issues if another protobuf is in the same python env.

Reply:
What's the proper fix?

Reply:
Move this to an python package override outside of pythonPackages

Reply:
Do you mean that I should add it to all-packages.nix or add it as a local override as I do?

Reply:
It should be used as a local overwrite in the package that requires it which must be outside of python-modules

Reply:
So I should move biliass out of python-modules?

Reply:
Ideally if it would be only a CLI which it isn't. Sigh. I am not sure what the best approach is here.

Reply:
We cannot handle multiple versions of a package in the same package set. Thus, if you *really* want it available as an importable library, it simply needs to reuse the protobuf used there. If that's not possible, then we just need to stick with the older version.

Reply:
> We cannot handle multiple versions of a package in the same package set.

Is this a bug? What's a package set?

> Thus, if you really want it available as an importable library, it simply needs to reuse the protobuf used there.

What's an importable library?

> If that's not possible, then we just need to stick with the older version.

Do you mean patch the source code to downgrade the lib? By the way, any idea why it suddenly failed to build?

Reply:
> Is this a bug?

No, a limitation of python.

> What's a package set?

eg python39Packages or python310Packages